packages/gitbook/src/components/DocumentView/Table/utils.ts

```diff
@@ -110,3 +110,7 @@ export function getColumnVerticalAlignment(column: DocumentTableDefinition): Ver
 
     return 'self-center';
 }
+
+export function isContentRef(ref: DocumentTableRecord['values'][string]): ref is ContentRef {
+    return Boolean(ref && typeof ref === 'object' && 'kind' in ref && 'file' in ref);
```

Comment:
```suggestion
    return Boolean(ref && typeof ref === 'object' && 'kind' in ref);
```

your current check only allows file refs to pass

Reply:
Yeah just realized that when i looked at the argos changes. I shouldn't have trusted copilot :smile: